test/Tests/src/Data/Bool_Spec.enso

```diff
@@ -0,0 +1,18 @@
+from Standard.Base import all
+
+import Standard.Test
+from Standard.Base.Data.Ordering import Equal, Less, Greater
+
+spec =
+    Test.group "Text" <|
+        Test.specify "should allow converting Bools to Text values" <|
+            True.to_text . should_equal "True"
+            False.to_text . should_equal "False"
+
+        Test.specify "should allow for comparing Bools" <|
```

Comment:
Surely this is the wrong group for this spec?

Reply:
yeap,thanks